tests/cypress.base.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     viewportHeight: 960,
     defaultCommandTimeout: 25000,
     downloadsFolder: 'cypress/fixtures',
+    chromeWebSecurity: false,
```

Comment:
Could you please clarify?

Reply:
Sometimes the test case_102.. crashes with an error:
```
SecurityError: Blocked a frame with origin "http://localhost:8080" from accessing a cross-origin frame.
```

https://github.com/cvat-ai/cvat/actions/runs/9309794034/job/25626028217#step:12:139

Reply:
And it seems that the chrome security works here

Reply:
I do not think we just want to disable web security in chrome
It would be a dirty fix.

Message: "Blocked a frame with origin "http://localhost:8080" from accessing a cross-origin frame." means that in tests from our code we are trying to make a request to another origin. 

But, actually, I do not remember we actually do this. So, need to understand what request causes the issue.

Reply:
Double checked this. 
We do not try to access other origins, but somehow Cypress runner and <iframe> with our app have different origins.. :/ 

Anyway, please try to sort out it instead of disabling web security at all.

Reply:
Yeah, I didn't find a request to another origin in this test either. I'll find out what request is causing this problem. I think it can be done by a separate pull-request. Now returned chromeWebSecurity to default value.